fiboa_cli/convert_utils.py

```diff
@@ -120,7 +122,7 @@ def normalize_geojson_properties(feature):
     return feature
 
 
-def read_geojson(path, **kwargs):
+def read_geojson(path, layer=None, **kwargs):
```

Comment:
The layer seems to be unused. What's the purpose?

Reply:
The [read_geojson() invocation](https://github.com/fiboa/cli/blob/spain/fiboa_cli/convert_utils.py#L307) is passed a `layer` keyword-arg, due to [the optional layer filtering feature](https://github.com/fiboa/cli/blob/spain/fiboa_cli/convert_utils.py#L292). This layer param avoids passing this `layer` kwarg to the [open() call in read_geojson()](https://github.com/fiboa/cli/blob/spain/fiboa_cli/convert_utils.py#L293). Alternatively, we could avoid filtering for layers in both parquet & geojson; I think this layer concept is not supported in geojson either.

Reply:
Yeah, layers is not really a thing in GeoJSON. Not even sure what that means. Does this filter on a layer property or so?
Rhis feels like a case where it should not be supported by the generic reader but be somehow implemented only in the specific converter by overriding a portion of the reader or so.

Reply:
No, ogr supports layers (because shapefile and gpkg support combining layers). But geojson files can't embed this, and GEOS makes a 'dummy' layer with the name of the file...